pkg/fluxexec/fluxexec_test.go

```diff
@@ -0,0 +1,32 @@
+package fluxexec
+
+import (
+	"bytes"
+	"context"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/weaveworks/weave-gitops/pkg/fluxinstall"
+	"github.com/weaveworks/weave-gitops/pkg/fluxinstall/src"
+)
+
+var _ = Describe("FluxExec", func() {
+	It("should be able to install Flux into the cluster", func() {
+		By("run flux install", func() {
```

Comment:
This is an integration test - it does end to end testing of downloading the binary, running the binary, and observing the side effects of doing that. But it will run as though it's a unit test - this test suite should not do network requests, and shouldn't really depend on the flux binary.

Both fluxinstall and flux.Install are both already tested. If this is meant to test that the binary invocation works, add e.g. a shell script that echoes something known and execute that.